cache.js

```diff
@@ -35,19 +34,16 @@ function branchPathExists (repoID, prID, branchTag) {
  * Save a file to local PR cache, distinguish by revision
  * @param {number} repoID unique repository identifier
  * @param {number} prID unique pull request identifier
- * @param {string} branchTag a tag to identify the current file revision
  * @param {string} filePath relative file path
  * @param {any} data file data
  * @param {string} fileType Optional: file type so it knows where to save the file (default python)
  * It's needed because go files should be in the GOPATH.
  */
-function saveFileToPRCache (repoID, prID, branchTag, filePath, data, fileType) {
+function saveFileToPRCache (repoID, prID, filePath, data, fileType) {
   fileType = fileType || 'python'
-  if (fileType === 'go') {
-    writeFileCreateDirs(path.join('cache/go/src', repoID.toString(), prID.toString(), filePath), data)
-  } else if (fileType === 'python') {
-    writeFileCreateDirs(path.join('cache', repoID.toString(), prID.toString(), branchTag, filePath), data)
-  }
+  const appTag = (fileType === 'go') ? 'gosec' : 'bandit'
```

Comment:
Should probably start using a const for 'gosec' and 'bandit'.  I've seen the strings more than once.  Probably from bandit.js and gosec.js would be appropriate.

Reply:
I agree but I'd rather tackle that as a follow up.
Opening #103